### ISSUE TITLE
Semantic tree problems with / division and invisible times

### ISSUE THREAD

**Issue by zorkow** [zorkow/speech-rule-engine#134]
The following expression gives a Math Processing error. Can't be further minimised.

```xml
 <math>
   <mrow>
      <mi>n</mi>
      <mo mathvariant="normal">/</mo>
      <mi>ϕ</mi>
    </mrow>
    <mo mathvariant="italic">⁢</mo>
    <mi>n</mi>
  </math>
```

**Comment by zorkow** [comment]
The invisible times comes from LaTeXML that adds invisible times incorrectly in places, where invisible function applications should go.

**Comment by zorkow** [comment]
Fixed in PR #146.
